src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -19269,40 +19269,69 @@ static void F4<T4>() where T4 : new()
         x4 = new T4?(); // warn 9
         x4 = new T4? { }; // warn 10
         x4 = (new T4?[1])[0];
+        x4 = new System.Nullable<int>? { }; // warn 11
+    }
+    static void F4<T4>() where T4 : class
```

Comment:
`F5<T5>() where T5 : class`

Reply:
Sorry, I should have paid more attention.